### PR TITLE
Add the documentation for the workOrders.download endpoint

### PR DESCRIPTION
**Pull request by JordyDeGroeve**

### CODE REVIEW

**Review by WouterSioen — APPROVED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -3713,6 +3713,25 @@ Send a finalized work order to the customer.
...
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `4afb0a9c-91c6-49ed-a2e5-ce7c1e3a87fb` (string, required)
+        + format: `pdf` (enum[string], required)
+            + Members
+                + pdf
+
++ Response 200 (application/json;charset=utf-8)
```
Comment (tiagobastos) [inline apiary.apib]: Didn't we remove the `charset` from the responses/requests?

**Review by tiagobastos — APPROVED** [review]

File: apiary.apib (hunk 1 above)
Comment (JordyDeGroeve) [reply apiary.apib]: Copied blindly from the proposal..  🤦‍♂️ 
Should we do this for all responses?

Comment (tiagobastos) [reply apiary.apib]: It's like that for master already. 
So I guess we should change that to all requests/responses in `work-orders` branch?

**Review by rathesDot — APPROVED** [review]

**Review by tiagobastos — APPROVED** [review]

**Review by tiagobastos — APPROVED** [review]

**Review by dietrichm — APPROVED** [review]

**Review by mrtus — APPROVED** [review]

**Review by WouterSioen — APPROVED** [review]

File: apiary.apib [hunk 2]
```diff
@@ -3723,13 +3723,32 @@ Finalize a draft work order. This will also generate the work order PDF.
...
+        + format: `pdf` (enum[string], required)
+            + Members
+                + pdf
+
++ Response 200 (application/json)
+
+    + Attributes (object)
+        + data (object)
+            + location: `https://cdn.teamleader.eu/file` (string) - A temporary url where the requested file can be downloaded
+            + expires: `2018-02-05T16:44:33+00:00` (string) - Expiration time of the temporary download link
```
Comment (jenssegers) [inline apiary.apib:3750]: Should be `expires_at` actually. Seems like we made this mistake for downloading invoices as well.

Comment (JordyDeGroeve) [reply apiary.apib:3750]: We'll need to discuss how to keep this consistent on all the `.download` endpoints.

**Pull request merged**